tools/data-handler/src/command-handler.ts

```diff
@@ -528,6 +590,57 @@ export class Commands {
     }
   }
 
+  /**
+   * Creates a new link
+   * @param cardKey Card key of the card where the link is created
+   * @param destinationCardKey Card key of the destination card
+   * @param linkType Name of the linktype
+   * @param linkDescription Description of the link
+   * @param path Optional, path to the project. If omitted, project is set from current path.
+   */
+  private async createLink(
+    cardKey: string,
+    destinationCardKey: string,
+    linkType: string,
+    linkDescription: string,
+    path: string,
+  ): Promise<requestStatus> {
+    try {
+      await this.createCmd.createLink(
+        path,
+        cardKey,
+        linkType,
+        destinationCardKey,
+        linkDescription,
+      );
+      return { statusCode: 200 };
+    } catch (e) {
+      return { statusCode: 400, message: errorFunction(e) };
+    }
+  }
+
+  /**
+   * Creates a new linktype.
+   * @param {string} name Name of the linktype.
```

Comment:
The other parameter is missing from command block

Reply:
Updated comment